Mathlib/NumberTheory/Cyclotomic/Basic.lean

```diff
@@ -493,7 +493,7 @@ variable {n S}
 /-- A cyclotomic extension splits `X ^ n - 1` if `n ∈ S`. -/
 theorem splits_X_pow_sub_one [H : IsCyclotomicExtension S K L] (hS : n ∈ S) :
     Splits (map (algebraMap K L) (X ^ n - 1)) := by
```

Comment:
This theorem should probably just say `Splits (X ^ n - 1 : L[X])` instead. But this is out of scope of this PR.